lib/AST/Module.cpp

```diff
@@ -34,6 +34,7 @@
 #include "swift/AST/ProtocolConformance.h"
 #include "swift/Basic/Compiler.h"
 #include "swift/Basic/SourceManager.h"
+#include "swift/Parse/Token.h"
```

Comment:
How is this better? Just fewer includes?

Reply:
Yeah, editing Token.h does not trigger recompilation of ~400 source files anymore.